Date/Span.php

```diff
@@ -1144,7 +1144,7 @@ function setDefaultFormat($format)
      * @static
      * @see      Date_Span::setDefaultFormat(), Date_Span::getDefaultInputFormat()
      */
-    function getDefaultFormat()
+    static function getDefaultFormat()
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.